### ISSUE TITLE
Improve on how VAPOR identifies VDC files

### ISSUE THREAD

**Issue by shaomeng** [NCAR/VAPOR#2984]
The latest VAPOR on main branch seems to lose its ability to recognize *.vdc files. Specifically, when passing a vdc file to VAPOR executable via command line, a window will pop up saying that "Could not determine dataset type."

There can be understandable confusion when an unknown sourced netCDF is passed to VAPOR, but given that the VDC format is defined by VAPOR and all VDC files are produced by VAPOR, any confusion could be removed by defining a unique VDC signature and recognizing that VDC signature when a file is passed in through command line.

**Comment by clyne** [comment]
Agreed. This should be a one-liner. VDC files define an attribute, VDC.Version, whose presence can be checked for.
